tests/framework/extensions/rke1/nodepools/nodepools.go

```diff
@@ -138,3 +141,82 @@ func ScaleNodePoolNodes(client *rancher.Client, cluster *management.Cluster, nod
 
 	return updatedNodePool, nil
 }
+
+// AddRKE1CustomClusterNodes is a helper method that will add nodes to the custom RKE1 custom cluster.
+func AddRKE1CustomClusterNodes(client *rancher.Client, cluster *management.Cluster, nodes []*nodes.Node, rolesPerNode []string) error {
```

Comment:
same thought around rke1, I think it would make more sense to have them in `provisioning/creates.go` or perhaps create a 'update.go` and 'delete.go`

A little tricky since these are happening per-node. But still, is part of the provisioning CRUD so would like to get more thoughts on this from others. Especially given the shared nature between the functions.